code/style.css

```diff
@@ -1,4 +1,359 @@
+/* 
 html {
   background: #0025ff;
   color: #fff;
+ } */
+
+body {
+  color: #082B48;
+  font-family: Vinyl, Segoe UI, San Francisco, Roboto;
+  line-height: 1.5;
+  margin: 0;
+  padding: 0;
 }
+
+img {
+  width: 100%;
+  height: auto;
+}
+
+a {
+  text-decoration: none;
+  color: #f0f0f0;
+}
+
+a:hover{
+  opacity: 0.7;
+}
+
+.container {
+  width: 90%;
+  max-width: 980px;
+  margin: auto;
+}
+
+ul {
+  list-style: none;
+  display: flex;
+}
+
+.clear::after {
+  content: "";
+  clear: both;
+  display:block;
+}
+
+* {
+  box-sizing:border-box;
+}
+
+/*======================
+ header 
+ =======================*/
```

Comment:
I really like that you are accurate about giving comments of each part/section before adding the specific css :-)